acceptance/bundle/deploy/secret-scope/test.toml

```diff
@@ -0,0 +1,75 @@
+Cloud = true
+Local = true
+
+Ignore = [
+    "databricks.yml",
+]
+
+[[Server]]
+Pattern = "POST /api/2.0/secrets/scopes/create"
```

Comment:
[optional] We have CRUD implementations in test server for various resources and it's not a lot of work but makes it usable for other tests. We definitely going to need it when we add tests for terraformless implementation.

https://github.com/databricks/cli/blob/main/acceptance/internal/handlers.go#L220
https://github.com/databricks/cli/tree/main/libs/testserver

Reply:
i suggest to do this in a follow-up PR